### PR TITLE
Fix: Nested Elements - Sync repeater with containers [ED-7545]

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

File: modules/nested-elements/assets/js/editor/nested-repeater/hooks/data/document/repeater/duplicate/nested-repeater-duplicate-container.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import Base from '../../../base';
+import { findChildContainerOrFaild } from 'elementor/modules/nested-elements/assets/js/editor/utils';
+
+export class NestedRepeaterDuplicateContainer extends Base {
+	getId() {
+		return 'nested-repeater-duplicate-container';
```
Comment (StyleShit) [inline modules/nested-elements/assets/js/editor/nested-repeater/hooks/data/document/repeater/duplicate/nested-repeater-duplicate-container.js:6]:
```suggestion
		return 'document/repeater/duplicate--nested-repeater-duplicate-container';
```

File: modules/nested-elements/assets/js/editor/nested-repeater/hooks/data/document/repeater/insert/nested-repeater-create-container.js [hunk 2]
```diff
@@ -16,51 +17,25 @@ export class NestedRepeaterCreateContainer extends Base {
 		return 'document/repeater/insert';
 	}
 
-	apply( args ) {
-		const { containers = [ args.container ] } = args,
-			component = $e.components.get( 'nested-elements/nested-repeater' );
+	getConditions( args ) {
+		// Will only handle when command called directly and not through another command like `duplicate` or `move`.
+		const isCommandCalledDirectly = $e.commands.isCurrentFirstTrace( 'document/repeater/insert' );
```
Comment (StyleShit) [inline modules/nested-elements/assets/js/editor/nested-repeater/hooks/data/document/repeater/insert/nested-repeater-create-container.js:22]: maybe use `this.getCommand()` for the parameter?

File: modules/nested-elements/assets/js/editor/nested-repeater/hooks/data/document/repeater/move/nested-repeater-move-container.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import Base from '../../../base';
+import { findChildContainerOrFaild } from 'elementor/modules/nested-elements/assets/js/editor/utils';
+
+export class NestedRepeaterMoveContainer extends Base {
+	getId() {
+		return 'nested-repeater-move-container';
```
Comment (StyleShit) [inline modules/nested-elements/assets/js/editor/nested-repeater/hooks/data/document/repeater/move/nested-repeater-move-container.js:6]: same as above regarding the ID

File: modules/nested-elements/assets/js/editor/nested-repeater/hooks/data/document/repeater/remove/nested-repeater-remove-container.js [hunk 4]
```diff
@@ -12,15 +13,17 @@ export class NestedRepeaterRemoveContainer extends Base {
 		return 'document/repeater/remove';
 	}
 
-	apply( args ) {
-		const { containers = [ args.container ] } = args;
+	getConditions( args ) {
+		// Will only handle when command called directly and not through another command like `duplicate` or `move`.
+		const isCommandCalledDirectly = $e.commands.isCurrentFirstTrace( 'document/repeater/remove' );
```
Comment (StyleShit) [inline modules/nested-elements/assets/js/editor/nested-repeater/hooks/data/document/repeater/remove/nested-repeater-remove-container.js:18]: same as above

File: modules/nested-elements/assets/js/editor/utils.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+	// Translations comes from server side.
+	return sprintf( title, index );
+}
+
+export function isWidgetSupportNesting( widgetType ) {
+	// eslint-disable-next-line camelcase
+	return elementor.widgetsCache[ widgetType ]?.support_nesting;
+}
+
+export function findChildContainerOrFaild( container, index ) {
```
Comment (StyleShit) [inline modules/nested-elements/assets/js/editor/utils.js:13]:
```suggestion
export function findChildContainerOrFail( container, index ) {
```

File: modules/nested-elements/assets/js/editor/utils.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+	const childView = container.view.children.findByIndex( index );
+
+	if ( ! childView ) {
+		throw new Error( 'Child container was not founded for the current repeater item.' );
```
Comment (StyleShit) [inline modules/nested-elements/assets/js/editor/utils.js:17]: found*

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: modules/nested-elements/assets/js/editor/nested-repeater/hooks/data/document/repeater/insert/nested-repeater-create-container.js (hunk 2 above)
Comment (Nevoss) [reply modules/nested-elements/assets/js/editor/nested-repeater/hooks/data/document/repeater/insert/nested-repeater-create-container.js:22]: Done

File: modules/nested-elements/assets/js/editor/nested-repeater/hooks/data/document/repeater/move/nested-repeater-move-container.js (hunk 3 above)
Comment (Nevoss) [reply modules/nested-elements/assets/js/editor/nested-repeater/hooks/data/document/repeater/move/nested-repeater-move-container.js:6]: Done

File: modules/nested-elements/assets/js/editor/nested-repeater/hooks/data/document/repeater/remove/nested-repeater-remove-container.js (hunk 4 above)
Comment (Nevoss) [reply modules/nested-elements/assets/js/editor/nested-repeater/hooks/data/document/repeater/remove/nested-repeater-remove-container.js:18]: Done

File: modules/nested-elements/assets/js/editor/utils.js (hunk 5 above)
Comment (Nevoss) [reply modules/nested-elements/assets/js/editor/utils.js:13]: Done

**Review by matipojo — CHANGES_REQUESTED** [review]
Please fix Lint errors

**Pull request merged**